src/fee-accounts/containers/account-summary/account-summary.component.html

```diff
@@ -10,5 +10,9 @@ <h1 class="govuk-body govuk-!-font-size-48 govuk-!-font-weight-bold">
   </div>
 </div>
 <app-prd-summary-component
+  *ngIf="(accountSummary$ | async).length === 0 else loading"
```

Comment:
i think this loading should be hooked on the reducer loading property?

Reply:
hmm, I thought I had this updated. Will update